test/math.jl

```diff
@@ -1396,6 +1398,9 @@ end
     # test for large negative exponent where error compensation matters
     @test 0.9999999955206014^-1.0e8 == 1.565084574870928
     @test 3e18^20 == Inf
+    # two cases where we have observed > 1 ULP in the past
+    @test 0.0013653274095082324^-97.60372292227069 == 4.088393948750035e279
+    @test 8.758520413376658e-5^70.55863059215994 == 5.052076767078296e-287
```

Comment:
Thanks for adding the tests!  Should be guarded by `has_fma[Float64]`, or we'd get the same exact result also on non-FMA CPUs?

Reply:
It should, but I haven't tested (don't have especially easy access to a computer without).

Reply:
Can't you start julia with a specific flag? like --target=ivy-bridge ?

Reply:
`fma` is emulated when not available directly from hardware. See `Base.fma_emulated` in `floatfuncs.jl`. The emulated version should be accurate, it's just very very slow.

Reply:
If `fma` is important to accuracy, do not use `muladd`. `muladd` does not consistently convert to `fma` (even when it's natively supported). `muladd` gives the option to the compiler, but I find that whether it actually uses `fma` is largely a coin toss. It's also subject to change with inlining, optimization level, and architecture.

In particular, any test that a function produces exact values while using `muladd` seems prone to unpredictable failures.

Reply:
I've fairly carefully validated that the `muladd`s here do good things on computers with `fma`. It helps that most of these `muladds` work either way. They are designed such that all of them (other than the ones on lines 212 and 213) won't significantly effect the results.

Reply:
can confirm that both of these tests pass without fma. (which is not that surprising)